core/ui/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/ui/AutoSizableText.kt

```diff
@@ -0,0 +1,64 @@
+package io.github.droidkaigi.confsched2023.ui
+
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableFloatStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalFontFamilyResolver
+import androidx.compose.ui.text.Paragraph
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.sp
+
+private const val CONTRACTION_RATIO = 0.95f
+
+/**
+ * ref: https://blog.canopas.com/autosizing-textfield-in-jetpack-compose-7a80f0270853
```

Comment:
Thanks for providing the references! Is this your original implementation? Otherwise, just to clarify, could you please tell us if this code complies with our license? Thanks!

Reply:
I don't think I can say that it is 100% original, as I am sure that I am referring to an article.
However, I would like to say that there are many parts that I came up with or researched and created myself, so there is no problem.

I don't see anything in the article (Medium) or in the sample code on GitHub about licensing, and I honestly don't know what to do if the above is insufficient.